django/contrib/admin/static/admin/js/autocomplete.js

```diff
@@ -0,0 +1,40 @@
+(function($) {
```

Comment:
Same here as for the un-minified CSS file. Why do we need to ship this one with Django?